packages/fxa-graphql-api/src/backend/profile-client.service.spec.ts

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { Provider } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import nock from 'nock';
+
+import { AuthClientService } from './auth-client.service';
+import { ProfileClientService } from './profile-client.service';
+import { ConfigService } from '@nestjs/config';
+
+describe('ProfileClientService', () => {
+  let service: ProfileClientService;
+  let authClient: any;
```

Comment:
It looks like you could eliminate the need to mock `createOAuthToken` each test by just doing

```tsx
  const authClient = {
    createOAuthToken: jest.fn().mockResolvedValue({ access_token: 'test' }),
  };
```